### PR TITLE
Install awscli

### PR DESCRIPTION
**Pull request by IliasDeros**
This will remove a step for the actual flatbook project setup

### CODE REVIEW

**Review by patvong — APPROVED** [review]

**Comment by patvong** [comment]
Nice, thanks @IliasDeros !

**Comment by IliasDeros** [comment]
Will merge by end of day unless someone objects to these trivial changes

File: mac [hunk 1]
```diff
@@ -128,9 +131,8 @@ brew install rbenv
 rbenv init
 echo 'eval "$(rbenv init -)"' >>~/.bash_profile
 
-
 fancy_echo "Installing latest Ruby..."
```
Comment (JonathanSchwartzSonder) [inline mac]: we should probably make this explicitly say which ruby it's installing.  Cause this is very much not the latest.

**Review by JonathanSchwartzSonder — APPROVED** [review]

File: mac (hunk 1 above)
Comment (IliasDeros) [reply mac]: Good point, this will be consistent with the Node installation message, too! Addressed.

**Pull request merged**